src/module.c

```diff
@@ -2027,7 +2027,7 @@ int RM_IsModuleNameBusy(const char *name) {
 }
 
 /* Return the current UNIX time in milliseconds. */
-long long RM_Milliseconds(void) {
+mstime_t RM_Milliseconds(void) {
```

Comment:
I was looking at some calls to mstime() and saw this.
do we also need a `RM_CachedMilliseconds` that will return `server.mstime`?

Reply:
One can use CachedMicroseconds and divide by 1000